lib/Smartcat/App/Command.pm

```diff
@@ -32,6 +32,12 @@ sub file_params_opt_spec {
     );
 }
 
+sub split_filename_by_triple_dash_opt_spec {
+    return (
+        [ 'split-filename-by-triple-dash' => 'Split filename by triple dash to <name> and <id>' ],
```

Comment:
I would want us to be backward-compatible in terms of parameter names in case we decide to customize or change the separator in the future. How about `extract-id-from-name`? And the description would become: `Extract stable external document identifiers from filenames in "name---id.ext" format`

Reply:
Yes, its seems better. Renamed:
```
sub extract_id_from_name_opt_spec {
    return (
        [ 'extract-id-from-name' => 'Extract stable external document identifiers from filenames in "name---id.ext" format for comparison and automatic renaming' ],
    );
}
```

PR renamed too